flashinfer/moe_ep/kernel_src/cutedsl_megamoe/src/moe_nvfp4_swapab/epilogue_refactor.py

```
@@ -921,20 +883,14 @@ def __init__(
-        self.bot = TmemTranspose16x32(
-            tmem_ptr, Region.Bottom, reg_tensor=reg_tensor_bot
-        )
-
+        self.bot = TmemTranspose16x32(tmem_ptr, Region.Bottom, reg_tensor=reg_tensor_bot)
+    
     def from_r1_perm_until_last_store(self) -> Tuple[cute.Tensor, cute.Tensor]:
-        self.bot.r1_perm()
-        self.top.r1_perm()
-        self.bot.r1_store()
-        self.top.r1_store()
+        self.bot.r1_perm();   self.top.r1_perm()
```

Ruff (0.15.21)

[error] Multiple statements on one line (semicolon)

(E702)